spec/fixtures/18ZOO/sr_power.midas.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
I just pushed e04c9112f to fix accessing these fixtures in the browser; this fixture fails to render at action 9

http://localhost:9292/fixture/18ZOO/sr_power.midas?action=8 works, then hitting next in the history controls breaks

Reply:
actually looks like more than of these fixtures are failing to render, with the same error:

![Screenshot 2021-03-20 215324](https://user-images.githubusercontent.com/1045173/111893225-c5344880-89c6-11eb-8e95-b59e148471c2.png)

Reply:
these are the others that fail:
* sr_power.too_much_responsibility
* sr_power.leprechaun_pot_of_gold
* sr_power.it_is_all_greek_to_me

Reply:
Confirmed fixtures are fixed by 28cff0187 👍